### DISCUSSION TITLE
Unable to create Android Package

### DISCUSSION THREAD

**Discussion by Shaunmax** [airsdk/Adobe-Runtime-Support#2701] [General]
Hello,

Today I am not able to create android package because I am getting the following error : 

`Warning: packagedDependency okio-1.17.5.jar used by more than one ANE. Ignoring this entry NOTE: Multiple Android build-tools folders exist, choosing the latest version (33.0.0 rc4) for packaging unexpected failure: Unable to run java: com.adobe.air.ADTException: gradle tool failed: Warning: SDK processing. package.xml parsing problem. unrecognized type name: platform. Did you mean archives? Warning: SDK processing. package.xml parsing problem. Unable to create an instance of com.android.repository.impl.generated.v2.TypeDetails java.lang.InstantiationException`

![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/16576562/194321fd-e5ca-47f9-8635-7c1f8e26f1a8)

I didn't make any update to the AIR sdk or files associated with it. Till last week everything was working fine -

**Comment by ajwfrost** [comment]
Possibly there's been an update of the Android Studio or gradle tools? If nothing changed with the AIR side then this is the other part that it uses. Assuming you don't have a file "package.xml" as part of your build (which could potentially confuse it!) you could perhaps try  adding the below lines into your app descriptor file within the `<android>` section (but outside of any `<manifestAdditions>`):
```
    <gradleVersion>7.5</gradleVersion>
    <androidGradlePluginVersion>7.4.0</androidGradlePluginVersion>
```

thanks

**Reply by Shaunmax** [reply]
@ajwfrost Unless there were some automatic updates, I dont think I did anything from my side. The last thing updated on my system related to app development was `apm`. 

I already have that in my app.xml :

```
<android>
    <containsVideo>false</containsVideo>
    <gradleVersion>7.5</gradleVersion>
    <androidGradlePluginVersion>7.4.0</androidGradlePluginVersion>
    <manifestAdditions><![CDATA[<manifest android:installLocation="auto" >
  <uses-sdk android:minSdkVersion="19" android:targetSdkVersion="31"/>
  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```

**Comment by Shaunmax** [comment]
I just updated the Android Studio and now everything is working fine!
